var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -110,7 +110,8 @@ class Rocblas(CMakePackage):
 
     depends_on("googletest@1.10.0:", type="test")
     depends_on("netlib-lapack@3.7.1:", type="test")
-    depends_on("llvm-amdgpu +openmp", type="test")
+    depends_on("llvm-amdgpu", type="test")
+    depends_on("rocm-openmp-extras", type="test")
```

Comment:
The `depends_on("llvm-amdgpu +openmp", type="test")` was just to add the `+openmp` when building the tests. Is `depends_on("llvm-amdgpu", type="test")` required to make OpenMP work? I would expect `depends_on("rocm-openmp-extras", type="test")` to be sufficient.